### PR TITLE
Add merged objects patch

### PR DESCRIPTION
**Pull request by IllusiveMan196**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -525,5 +525,13 @@ plugins:
       - 'Graphic Herbalism TR Extra.ESP'
 
 #Any dynamic patches (not sure if multipatch is the  only one)
+  - name: 'Merged Objects.esp'
```
Comment (MacSplody) [inline masterlist.yaml:528]: You could add the location for the mod.

```suggestion
  - name: 'Merged Objects.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/46870/' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -525,5 +525,13 @@ plugins:
...
+    group: *dynamicPatchGroup
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenrate after adding or removing mods changing or adding in-game items. Use TES3Merge'
+            url: [https://www.nexusmods.com/morrowind/mods/46870]
```
Comment (MacSplody) [inline masterlist.yaml]: For links in messages use `[example](https://www.example.com/)`.
Add a newline between separate mods as well.

```suggestion
            text: 'Regenerate after adding or removing mods changing or adding in-game items. Use [TES3Merge](https://www.nexusmods.com/morrowind/mods/46870/)'

```

> Regenerate after adding or removing mods changing or adding in-game items.

Wording could be changed slightly.

> Regenerate after adding or removing mods that change or add in-game items.

**Review by MacSplody — CHANGES_REQUESTED** [review]
Two changes required, and one other suggestion.

**Comment by IllusiveMan196** [comment]
Added proposed changes, also more logical order of patches.

File: masterlist.yaml [hunk 3]
```diff
@@ -226,7 +226,7 @@ common: []
 bash_tags: []
 
 globals:
-# Latest LOOT Thread
+  # Latest LOOT Thread
```
Comment (MacSplody) [inline masterlist.yaml]: Not related to adding merged objects, so it should be done in a separate pull request,
```suggestion
# Latest LOOT Thread
```

File: masterlist.yaml [hunk 4]
```diff
@@ -247,15 +247,15 @@ groups:
     after: [ default ]
 
 plugins:
-# Official DLC
+  # Official DLC
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
# Official DLC
```

File: masterlist.yaml [hunk 5]
```diff
@@ -247,15 +247,15 @@ groups:
...
   - name: 'Tribunal.esm'
     group: *dlcGroup
 
   - name: 'Bloodmoon.esm'
     group: *dlcGroup
     after: [ 'Tribunal.esm' ]
 
-# Official Mods
+  # Official Mods
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
# Official Mods
```

File: masterlist.yaml [hunk 6]
```diff
@@ -401,7 +401,7 @@ plugins:
     url: [ 'https://www.nexusmods.com/morrowind/mods/43931' ]
     group: *earlyLoadersGroup
 
-# Unofficial Patches
+  # Unofficial Patches
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
# Unofficial Patches
```

File: masterlist.yaml [hunk 7]
```diff
@@ -495,7 +495,7 @@ plugins:
         subs: [ '"Script Improvements" version' ]
         condition: 'many("gr_ScriptImprovements.es(m|p)")'
 
-#Other Mods (default)
+  #Other Mods (default)
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
#Other Mods (default)
```

File: masterlist.yaml [hunk 8]
```diff
@@ -524,6 +524,22 @@ plugins:
       - 'Graphic Herbalism TR.ESP'
       - 'Graphic Herbalism TR Extra.ESP'
 
-#Any dynamic patches (not sure if multipatch is the  only one)
+  #Any dynamic patches
```
Comment (MacSplody) [inline masterlist.yaml]: Kind of related, but we should match section name and formatting from the other masterlists.

```suggestion
###### Modding Tools - Generated Files ######
```

File: masterlist.yaml [hunk 9]
```diff
@@ -524,6 +524,22 @@ plugins:
...
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenerate after adding or removing mods that change or add in-game items. Use [TES3Merge](https://www.nexusmods.com/morrowind/mods/46870/)'
+
   - name: 'multipatch.esp'
     group: *dynamicPatchGroup
+    after:
+      - 'Merged Objects.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Single entry lists can be placed in brackets.
```suggestion
    after: [ 'Merged Objects.esp' ]
```

File: masterlist.yaml [hunk 10]
```diff
@@ -524,6 +524,22 @@ plugins:
...
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Regenerate after adding or removing mods or generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/)'
```
Comment (MacSplody) [inline masterlist.yaml]: Missing newline at the end of file.

image

```suggestion
            text: 'Regenerate after adding or removing mods or generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/)'

```

**Review by MacSplody — CHANGES_REQUESTED** [review]

File: masterlist.yaml [hunk 11]
```diff
@@ -484,7 +484,7 @@ plugins:
       - name: 'Patch for Purists - Expansion Implementation.esm'
       - name: 'Patch for Purists - Dialogue Inconsistencies.esm'
 
-# Other Fixes
+  # Other Fixes
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
# Other Fixes
```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@IllusiveMan196 Thanks.

**Pull request merged**